optimade/validator/validator.py

```diff
@@ -470,6 +490,19 @@ def test_page_limit(self, response):
             raise ResponseError(
                 f"Endpoint did not obey page limit: {num_entries} entries vs {self.page_limit} limit"
             )
+
+        more_data_available = response.json()["meta"]["more_data_available"]
```

Comment:
Use `response` directly - is true for all subsequent lines here.
Possibly use the `.get` method as well to not raise an `AttributeError` or put it all in a `try/except`?

Reply:
Fair point, I guess I'm expecting everything to have worked by this point.

Reply:
On second thoughts, I feel checking these keys adds some extra validation, if they raise an then that is a useful piece of info about the response. I thought the `test_case` decorator would take any error raised by the underlying function and turn it into a `ResponseError`, but it looks like I wrote it to just accept `ResponseError`s, so I'll make sure they get raised with an informative message.

Reply:
Having `test_case` wrap all exceptions is not a bad idea either 👍

Reply:
Trouble then is that we could right garbage test functions that report as errors of the implementation, which feels a bit unfair on them :P At least this way someone can easily complain and see what's going on.